app/controllers/adjustments_controller.rb

```diff
@@ -1,26 +1,33 @@
 class AdjustmentsController < ApplicationController
   before_action :set_adjustment, only: %i[ show edit update destroy ]
+  after_action :verify_policy_scoped, only: [:index]
+  after_action :verify_authorized, except: [:index]
 
   # GET /adjustments or /adjustments.json
   def index
-    @adjustments = current_user.adjustments.all
+    #@adjustments = current_user.adjustments.all
+    @adjustments = policy_scope(Adjustment)
   end
 
   # GET /adjustments/1 or /adjustments/1.json
   def show
+    authorize @adjustment
   end
 
   # GET /adjustments/new
   def new
+    authorize Adjustment
```

Comment:
If you are going to keep the `new` action as `true` you can still pass the new record to the `authorize` method.

```suggestion
    @adjustment = Adjustment.new
    authorize @adjustment
```

Reply:
This makes sense.  I will add this to both the `loan` and `adjustment` controller. Thanks!